instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -96,6 +96,8 @@ def add(x, y):
 _TASK_REVOKED_TERMINATED_SIGNAL_KEY = "celery.terminated.signal"
 _TASK_NAME_KEY = "celery.task_name"
 
+_QUEUE_NAME = "queue"
```

Comment:
Why not celery? https://opentelemetry.io/docs/specs/semconv/attributes-registry/messaging/#messaging-system

Reply:
my bad, updating it

Reply:
done, also added tests.